src/datasets/loaders/10x_datasets/config.vsh.yaml

```diff
@@ -0,0 +1,96 @@
+functionality:
+  name: 10x_datasets
+  namespace: datasets/loaders
+  description: |
+    Download a SpaceRanger h5 gene expression file and spatial imaging data from the 10x genomics website (or someplace else).
+  
+  argument_groups: 
+    - name: Inputs
+      arguments:
+        - name: "--input_expression"
+          type: string
+          description: URL to the feature / barcode matrix HDF5.
+          required: true
+        - name: "--input_spatial"
+          type: string
+          description: URL to the Spatial imaging data.
```

Comment:
Improve the descriptions and examples so it's clear that this should be two urls of the same dataset